src/test/SelectionInterface.test.ts

```diff
@@ -331,70 +335,14 @@ describe("SelectionInterface module", () => {
                     { name: "A", tags: [""] },
                     { name: "C", tags: ["", "t1", "t2"] },
                 ],
-                colorChangeAgents: [],
+                colorChanges: initialColorChanges,
             };
             const ids = si.getHiddenIds(selectionStateHide);
 
             expect(ids).toEqual([0, 8, 9, 10, 11]);
         });
     });
 
-    describe("getColorChangeIds", () => {
-        test("Color change: select multiple by name", () => {
-            const si = new SelectionInterface();
-            si.parse(idMapping);
-            const selectionStateColor = {
-                highlightedAgents: [],
-                hiddenAgents: [],
-                colorChangeAgents: [
-                    { name: "A", tags: [] },
-                    { name: "C", tags: [] },
-                ],
-            };
-            const ids = si.getColorChangeAgentIds(
```

Comment:
this was really just testing the same code we already had tested now that I consolidated it into one function